agents/src/ipc/proc_pool.ts

```diff
@@ -22,12 +23,12 @@ export class ProcPool {
 
   constructor(
     agent: string,
-    numIdleProcesses: number,
+    _numIdleProcesses: number,
```

Comment:
can you keep this as the same name and just add an eslint-disable to this line? i don't want to break the API here since i'm going to fix this later

Reply:
This is not API breaking as it's not defined as an object. TS doesn't care about parameter names